src/admission_response_handler/policy_mode.rs

```diff
@@ -0,0 +1,19 @@
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug, Clone, Default, PartialEq)]
+pub enum PolicyMode {
+    #[serde(rename = "monitor")]
+    Monitor,
+    #[serde(rename = "protect")]
+    #[default]
+    Protect,
+}
```

Comment:
Let's use the [#[serde(rename_all = "...")]](https://serde.rs/container-attrs.html#rename_all)

Reply:
good point, done.